calico/_data/versions.yml

```diff
@@ -1,36 +1,36 @@
-- title: v3.24.0-pre
+- title: v3.24.0
   note: ""
   manifests_url: https://raw.githubusercontent.com/projectcalico/calico/master
```

Comment:
need to change this manifests_url as well

should be /v3.24.0 instead of /master